Zend/zend_inheritance.c

```diff
@@ -875,7 +875,8 @@ static void ZEND_COLD emit_incompatible_method_error(
 
 		if (!return_type_will_change_attribute) {
 			zend_error_at(E_DEPRECATED, NULL, func_lineno(child),
-				"Declaration of %s should be compatible with %s",
+				"Return type of %s should be either compatible with %s, "
+				"or the #[ReturnTypeWillChange] attribute should be used to suppress the notice",
```

Comment:
```suggestion
				"or the #[ReturnTypeWillChange] attribute should be used to temporarily suppress the notice",
```
Make it clearer that it's not a long-term solution?

Reply:
Thanks for the suggestions, they both make sense to me.